detectable/src/main/java/com/blackduck/integration/detectable/detectables/opam/parse/OpamFileParser.java

```diff
@@ -93,6 +95,15 @@ public Map<String, String> parseData(List<String> lines) {
         return output;
     }
 
+    private void addProjectNameAndVersion(String line, Map<String, String> output, Pattern pattern, String parsingValue) {
+        String value = line.split(":")[1];
+        Matcher matcher = pattern.matcher(value);
+        if (matcher.find()) {
+            value = matcher.group(1);
+        }
```

Comment:
No else {} block?

Reply:
If we cannot find the package name correctly, we leave it blank.